packages/core/core/src/utils/context/index.ts

```diff
@@ -10,11 +10,23 @@ const configureContext = (config: ContextConfiguration) => {
   useVSFContext = config.useVSFContext || useVSFContext;
 };
 
-const applyContextForApi = (api, context) =>
+const applyContextForApi = (api, context, extensions = []) =>
```

Comment:
https://github.com/kettanaito/naming-cheatsheet#naming-functions

are you applying context _for_ api or _to_ API?